We do not have the typed-ast package in the Userland yet. Since astroid can
work without it, we can safely remove it from its dependencies.

lazy_object_proxy is pinned for no good reason preventing us from updating it.

This was already fixed upstream:
https://github.com/PyCQA/astroid/commit/744a9a802543a46ff4c9e0727d8ea4fc83c80691

--- astroid-2.4.2/astroid.egg-info/requires.txt
+++ astroid-2.4.2/astroid.egg-info/requires.txt
@@ -1,6 +1,3 @@
-lazy_object_proxy==1.4.*
+lazy_object_proxy>=1.4.0
 six~=1.12
 wrapt~=1.11
-
-[:implementation_name == "cpython" and python_version < "3.8"]
-typed-ast<1.5,>=1.4.0
--- astroid-2.4.2/astroid/__pkginfo__.py
+++ astroid-2.4.2/astroid/__pkginfo__.py
@@ -26,10 +26,9 @@ numversion = tuple(int(elem) for elem in
 
 extras_require = {}
 install_requires = [
-    "lazy_object_proxy==1.4.*",
+    "lazy_object_proxy>=1.4.0",
     "six~=1.12",
     "wrapt~=1.11",
-    'typed-ast>=1.4.0,<1.5;implementation_name== "cpython" and python_version<"3.8"',
 ]
 
 # pylint: disable=redefined-builtin; why license is a builtin anyway?
